setup.py

```diff
@@ -42,7 +42,8 @@
                            'data/summarize_empty/empty_forward_in_paired/*',
                            'data/summarize_empty/empty_reverse_in_paired/*',
                            'data/summarize_empty/empty_paired_end/*',
-                           'data/reverse_only/*'
+                           'data/reverse_only/*',
+                           'data/*'
```

Comment:
Are these from a prior iteration where we had the usage data in the repo? Otherwise lgtm

Reply:
good catch @Oddant1, we don't need the data/* path since everything is in subdirectories. just removed that!